src/payload.rs

```diff
@@ -25,44 +25,35 @@ use {
     base64::prelude::{Engine, BASE64_STANDARD},
     serde::{Deserialize, Serialize},
     solana_client::rpc_config::RpcSendTransactionConfig,
-    solana_sdk::transaction::VersionedTransaction,
+    solana_sdk::{pubkey::Pubkey, transaction::VersionedTransaction},
     solana_transaction_status::UiTransactionEncoding,
     std::str::FromStr,
     thiserror::Error,
 };
 
 #[derive(Debug, Default, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct RpcSendTransactionConfigWithBlockList {
+pub struct RpcSendTransactionConfigWithForwardingPolicies {
     #[serde(flatten)]
-    pub config: Option<RpcSendTransactionConfig>,
-    /// base58-encoded pubkeys
-    pub blocklist_pdas: Option<Vec<String>>,
+    pub config: RpcSendTransactionConfig,
+    pub forwarding_policies: Vec<Pubkey>,
 }
 
-impl RpcSendTransactionConfigWithBlockList {
-    /// Converts the optional string-based blocklist_pdas into a Vec<Pubkey>
-    /// Invalid pubkeys are filtered out
-    pub fn blocklist_pubkeys(&self) -> Vec<solana_sdk::pubkey::Pubkey> {
-        match &self.blocklist_pdas {
-            Some(keys) => keys
-                .iter()
-                .filter_map(
-                    |key_str| match solana_sdk::pubkey::Pubkey::from_str(key_str) {
-                        Ok(pubkey) => Some(pubkey),
-                        Err(_) => None,
-                    },
-                )
-                .collect(),
-            None => Vec::new(),
-        }
-    }
+impl RpcSendTransactionConfigWithForwardingPolicies {
+    pub fn new(
+        config: Option<RpcSendTransactionConfig>,
+        forwarding_policies: Option<Vec<String>>,
+    ) -> Self {
+        let config = config.unwrap_or_default();
+        let forwarding_policies = forwarding_policies
+            .unwrap_or_default()
+            .iter()
+            .filter_map(|key| solana_sdk::pubkey::Pubkey::from_str(key).ok())
+            .collect::<Vec<Pubkey>>();
```

Comment:
@leafaar this is where the `Vec<String>` are converted to pubkey replace the `blocklist_pubkeys` method